embulk-standards/src/main/java/org/embulk/standards/JsonFormatterPlugin.java

```diff
@@ -0,0 +1,168 @@
+
+        // create a file
+        encoder.nextFile();
+
+        return new PageOutput() {
+            private final PageReader pageReader = new PageReader(schema);
+            public void add(Page page)
+            {
+                pageReader.setPage(page);
+                final ObjectMapper objectMapper = new ObjectMapper();
+                while (pageReader.nextRecord()) {
+                    final LinkedHashMap<String, Object> line = new LinkedHashMap<>();
+                    schema.visitColumns(new ColumnVisitor() {
+                        public void booleanColumn(Column column)
+                        {
+                            if (!pageReader.isNull(column)) {
+                                addValue(column.getName(), pageReader.getBoolean(column));
+                            } else {
+                                addNull(column.getName());
+                            }
+                        }
+
+                        public void longColumn(Column column)
+                        {
+                            if (!pageReader.isNull(column)) {
+                                addValue(column.getName(), pageReader.getLong(column));
+                            } else {
+                                addNull(column.getName());
+                            }
+                        }
+
+                        public void doubleColumn(Column column)
+                        {
+                            if (!pageReader.isNull(column)) {
+                                addValue(column.getName(), pageReader.getDouble(column));
+                            } else {
+                                addNull(column.getName());
+                            }
+                        }
+
+                        public void stringColumn(Column column)
+                        {
+                            if (!pageReader.isNull(column)) {
+                                addValue(column.getName(), pageReader.getString(column));
+                            } else {
+                                addNull(column.getName());
+                            }
+                        }
+
+                        public void timestampColumn(Column column)
+                        {
+                            if (!pageReader.isNull(column)) {
+                                Timestamp value = pageReader.getTimestamp(column);
+                                addValue(column.getName(), timestampFormatters[column.getIndex()].format(value));
+                            } else {
+                                addNull(column.getName());
+                            }
+                        }
+
+                        public void jsonColumn(Column column)
+                        {
+                            if (!pageReader.isNull(column)) {
+                                Value value = pageReader.getJson(column);
+                                addValue(column.getName(), value.toJson());
```

Comment:
this is unexpected because json value becomes a string.
For example, if a record has two columns ("time:timestamp" and "params:json"), and values are `time=2016-01-01 00:00:00`, `params={"q":"test"}`, a generated file should be: `{"time":"2016-01-01T00:00:00Z","params":{"q":"test"}`. It shouldn't be: `{"time":"2016-01-01T00:00:00Z","params":"{\"q\":\"test\"}"`.